src/gui/mainwindow.cpp

```diff
@@ -1352,6 +1359,12 @@ void MainWindow::on_actionTop_tool_bar_triggered()
     Preferences::instance()->setToolbarDisplayed(is_visible);
 }
 
+void MainWindow::on_actionShow_Status_bar_triggered() {
+    bool is_visible = static_cast<QAction*>(sender())->isChecked();
+    QMainWindow::statusBar()->setVisible(is_visible);
```

Comment:
Without `QMainWindow::` a reference to 'statusBar' is ambiguous (compile error).